contracts/internals/tokenWhitelist.sol

```diff
@@ -163,7 +165,8 @@ contract TokenWhitelist is ENSResolvable, Controllable, Ownable, Claimable {
 
     //// @notice Withdraw tokens from the smart contract to the specified account.
     function claim(address payable _to, address _asset, uint _amount) external onlyOwner {
```

Comment:
This function seems to be used in multiple places, should it be made into a contract object?
Same with the `Claimed` event.

Reply:
that is where we started, but i thought it would be better to generalise it is a safeTransfer contract ...